### DISCUSSION TITLE
Thank you so much for this amazing library! ❤️

### DISCUSSION THREAD

**Discussion by its-monotype** [ppetzold/nestjs-paginate#312] [General]
As the title says, we are all very grateful for your work, we hope that you will continue to develop/maintain this library, it really makes life easier and it's just great. I hope you continue to delight the Nest community with great libraries like this one. With best wishes and regards! Thanks again!

**Comment by ppetzold** [comment]
Hi there, thanks for the kind words. Glad people find this pagination helper useful. Also shout-out to all the other contributors.
